soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -2832,8 +2834,8 @@ void KaleidoScope_InitVertices(PlayState* play, GraphicsContext* gfxCtx) {
         }
     }
 
-    for (phi_t3 = 0; phi_t3 < 7; phi_t3++) {
-        phi_t4 = D_8082B11C[phi_t3];
+    for (phi_t3 = 0; phi_t3 < (CVarGetInteger("gBetterAmmoRendering", 0) ? ARRAY_COUNT(D_8082B11C_all) : ARRAY_COUNT(D_8082B11C)); phi_t3++) {
+        phi_t4 = CVarGetInteger("gBetterAmmoRendering", 0) ? D_8082B11C_all[phi_t3] : D_8082B11C[phi_t3];
```

Comment:
Same thing here about moving the CVar fetch out of the loop.

Reply:
Done